src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Windows/Markup/UsableDuringInitializationAttribute.cs

```diff
@@ -2,23 +2,16 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
-
-using System.Text;
-
```

Comment:
This will conflict with PR #103 - https://github.com/dotnet/wpf/pull/103/files#diff-6444baf87993ddeb8fd24c4b600ec744